client/src/components/PacmanGame/Cell.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { Rect } from 'react-konva';
+
+const Cell = ({ x, y, gridSize, entity, hasPacman }) => {
+  let color;
+  switch (entity) {
+    case 'pacman':
+      color = 'yellow';
```

Comment:
Refactor this to object key method Instead of switch, it is shorthand for switch case and will also remove mutation for color variable